src/components/frame/v5/pages/StreamingPaymentsPage/partials/NoExtensionBanner/NoExtensionBanner.tsx

```diff
@@ -0,0 +1,44 @@
+import { Extension } from '@colony/colony-js';
+import { WarningCircle } from '@phosphor-icons/react';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { COLONY_EXTENSIONS_ROUTE } from '~routes';
+import NotificationBanner from '~v5/shared/NotificationBanner/NotificationBanner.tsx';
+
+const displayName = 'v5.pages.StreamingPaymentsPage.partials.NoExtensionBanner';
+
+const MSG = defineMessages({
+  bannerInfo: {
+    id: `${displayName}.bannerInfo`,
+    defaultMessage: 'Streaming payments extension is currently not enabled.',
+  },
+  link: {
+    id: `${displayName}.link`,
+    defaultMessage: 'View extension',
+  },
+});
+
+const NoExtensionBanner = () => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <div className="pb-9">
+      <NotificationBanner
+        status="warning"
+        icon={WarningCircle}
+        callToAction={
+          <a href={`${COLONY_EXTENSIONS_ROUTE}/${Extension.StreamingPayments}`}>
+            {formatMessage(MSG.link)}
+          </a>
```

Comment:
By using the `<a>` tag, we're going to trigger a whole-page reload

https://github.com/user-attachments/assets/954947b6-1078-42bf-a63b-28d1b3c96f0b

Let's refactor this and use the `Link` component.

Also for the extension url we could use `const { extensionUrl } = useExtensionItem(Extension.StreamingPayments);`